subsys/bluetooth/host/hci_core.c

```diff
@@ -5208,23 +4872,24 @@ static const struct event_handler normal_events[] = {
 		      sizeof(struct bt_hci_evt_conn_request)),
 	EVENT_HANDLER(BT_HCI_EVT_CONN_COMPLETE, conn_complete,
```

Comment:
@jhedberg I was thinking of continuing the pattern below to move other parts out, for example conn_complete here can be moved to conn.c as well as other conn related events. And adv/scan can be similarly extracted, although I would like periodic advertising in first. Do you see this as a good way forward?

Reply:
Sure, sounds like a good plan

Reply:
Yes, this makes sense, +1 to that.